src/customer.ts

```diff
@@ -284,22 +285,25 @@ export class Customer extends ServiceFactory {
     }
 
     try {
+      const _parser = (rows: services.IGoogleAdsRow[]) => {
```

Comment:
this library is camel case only, so could we call this `parseWrapper` or something?

Reply:
👍